### ISSUE TITLE
[BUG] Noclip keep enabling and disabling hooks which kill framerate

### ISSUE THREAD

**Issue by xiaoxiao921** [harbingerofme/DebugToolkit#188]
**Describe the bug**
It also spam the console info level

**Does the bug happen without input**
no

**If input is required, which scenes does it go wrong in**
in a run while using noclip

**Do you have other mods installed?**
No

**Comment by SChinchi** [comment]
I can't replicate this with both a clean profile and the current dev profile I normally work with. Both for the released 3.16.2 version and the unreleased 3.16.3 on the repo.

What you describe though I had encountered once when I was modifying the command, but I don't see how it'd be triggered in the current version. I've enabled noclip and then used `spawn_as`, `respawn` and `next_stage` and in all cases there were no nulls. Both for survivors and flyer monsters.

Would it be possible to have a look at the dll that caused this and the il cursor line of the error?
